### DISCUSSION TITLE
Github page is down despite Githubs assessment that everything is working

### DISCUSSION THREAD

**Discussion by kogorman** [community/community#35771] [Other Feature Feedback, Questions, & Ideas]
The site https://cleeff.github.io/hex/ gives me an error report on a machine where I use it regularly.

I suspect this is a new problem, and I don't see a way to report it.

**Comment by jfullstackdev** [comment]
you might want to check the error logs and fix it

![image](https://user-images.githubusercontent.com/47092464/195122439-3bbb13b7-170c-452f-910d-7e9c5abc6d9c.png)
